<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0">

  <!-- Title: Enter Name -->
  <title>Ryan Kellogg - About</title>

  <!-- CSS Imports -->
  <link href="css/css/bootstrap.min.css" rel="stylesheet" type="text/css">
  <link href="css/css/font-awesome.min.css" rel="stylesheet" type="text/css">
  <link href="css/css/animate.css" rel="stylesheet" type="text/css">
  <link href="css/css/component.css" rel="stylesheet" type="text/css">
  <link href="css/css/simpletextrotator.css" rel="stylesheet" type="text/css">
  <link href="css/css/theme.css" rel="stylesheet" type="text/css">
  <link href="css/css/main.css" rel="stylesheet" type="text/css">
  <script src="js/modernizr.custom.js"></script>
</head>
<body>

  <!-- Header. Profile picture, name, tagline, resume download, and navbar. -->
  <header id="intro">
    <nav class="navbar navbar-default navbar-fixed-top" role="navigation">
      <div class="container">
        <div class="navbar-header">
          <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#myNavigation">
            <i class="fa fa-bars"></i>
          </button>
          <!--<img src="lib/img/IMG_6814.jpg" class="navbar-logo pull-left" alt="" />-->
          <a href="#intro" class="navbar-brand animated flipInX">RK</a>
        </div>

        <div class="collapse navbar-collapse" id="myNavigation">
          <ul class="nav navbar-nav navbar-right animated flipInX">
            <li><a href="#about" class="scroll">About</a></li>
            <li><a href="#outreach" class="scroll">Outreach</a></li>
            <li><a href="#publications" class="scroll">Publication</a></li>
          </ul>
        </div>
      </div>
    </nav>

    <!--
    <div class="jumbotron">
      <img src="lib/img/IMG_7292.jpg" class="img-responsive" alt="" />
       <h2 class="jumbo-name wow bounceInUp">Ryan Kellogg</h2> -->

      <!-- <p classs = "tagline"> 
      <a class="resume" href="">Resume</a> -->
    </div>
  </header>

<!--
 <div class="section" id="about">
    <div class="container">
      <div class="row">
        <div class="col-sm-3">
          <img src="lib/img/thumbs/heltberg2016_thumb.jpg" class="img-responsive img-rounded" align="right" alt="" />
        </div>
      </div>
    </div>
  </div>

<div class="row">
    <div class="col-sm-4">
        <div style="height:10em;border:1px solid #333"><img src="lib/img/thumbs/heltberg2016_thumb.jpg" class="img-responsive" align="right"></div>
    </div>
    <div class="col-sm-8">
        <div style="height:3em;border:1px solid #F00">Small</div>
    </div>
</div>
-->

  <!-- About. Short description of yourself, profile pic, experience -->
  <div class="section" id="about">
    <div class="container">
      <div class="row">
        <div class="col-sm-3" style="border:1px"></div>
        <div class="col-sm-9"><p class="title">Ryan Kellogg</p></div>
        <hr>
      </div>
      <div class="row">
        <div class="col-sm-3">
          <img src="lib/img/me_fb.jpg" align="right" class="img-responsive img-rounded" alt="" />
        </div>

        <div class="col-sm-9">
          
          <p class="description ">I am a postdoctoral fellow at Stanford University jointly advised by Profs. Michael Snyder and Stephen Quake (Genetics and Bioengineering departments, respectively). My work unites engineering and biology to understand how cells process information and make decisions. During my PhD I studied noise and digital activation in the NF-&kappa;B system using microfluidics and mathematical modeling. I currently work on how the microbiome communicates with the immune system. </p>
          
          <!--
          <p class="description "><a href="#">Snyder Lab website</a></p>
          <p class="description "><a href="#">Quake Lab website</a></p>        
         -->
         
        </div>
      </div>
    </div>
  </div> 


  <!-- Inbetween: Using quotes to de-clutter the webpage
  <div class="section quote-section">
    <div class="container">
      <div class="row">
        <div class="col-sm-6 col-sm-offset-3">
          <blockquote class="blockquote-reverse">
            <p><i class="fa fa-quote-left"></i> We keep moving forward, opening new doors, and doing new things, because we're curious and curiosity keeps leading us down new paths.</p>
            <p class="quoter"> - Walt Disney</p>
          </blockquote>
        </div>
      </div>
    </div>
  </div> 
 -->

  <!-- Expertise: Displays work experience -->
  <div class="section" id="education">
    <div class="container">
      <div class="row">
        <div class="col-sm-3"></div>
        <div class="col-sm-9">

          <p class="title">Education</P>
          <!-- <p class="subtitle">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce in turpis quam. Nulla quis eleifend lectus. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Mollitia, nihil fugiat id animi, corporis placeat perspiciatis at voluptatem quas culpa, quisquam labore atque? Aut alias, laboriosam blanditiis veritatis aliquid labore?</p> -->
        </div>
      </div>

      <div class="education">
        <div class="row">
          <div class="col-sm-3">
            <img src="lib/img/eth_logo.png" class="img-responsive center-block" width=220px>
          </div>
          <div class="col-sm-9">
            <h3><p>ETH Zurich</p></h3>
            <h4>
            <p>PhD "Dynamic Signal Processing in Gene Regulation"</p>
            <p>Laboratory of Prof. Savas Tay</p>
            <p>Whitaker fellowship in lab of Mike White at University of Manchester</p>
            <p></p>
            </h4>
          </div>
        </div>
        <div class="row">
          <div class="col-sm-3">
            <img src="lib/img/cmu_logo.png" class="img-responsive center-block" width=120px>
          </div>
          <div class="col-sm-9">
            <h3><p>Carnegie Mellon University</p></h2>
            <h4>
            <p>B.S. Electrical and Computer Engineering, 2007</p>
            <p>B.S. Biomedical Engineering, 2007 </p>
            <p>M.S. Engineering and Technology Innovation Management, 2008</p>
            <p>M.S. Biomedical Engineering, 2010 </p>
            </h4>
          </div>
        </div>
      </div>
    </div>
  </div> 


  <div class="section" id="outreach">
    <div class="container">
      <div class="row">
        <div class="col-md-12">
          <p class="title">Outreach</p>
          <h3><a href="http://stanfordbiosciencelima2017.wordpress.com">Stanford/EduExplora Bioscience class in Lima, Peru 2017</a></h3>
        </div>
      </div>
    </div>
  </div>

  <!-- Projects: Using Tympanus grid package to display side-projects -->
  <div class="section" id="publications">
    <div class="container">
      <div class="row">
        <div class="col-md-12">
          <h2 class="title">Publications</h2>
          <p class="subtitle">Click below to preview each paper.</p>
        </div>

        <div class="col-md-12">
          <ul id="og-grid" class="og-grid">
            <li>
              <a href="lib/ppr/heltberg2016.pdf" data-largesrc="lib/img/heltberg2016.jpg" data-title="Noise Induces Hopping between NF-&kappa;B Entrainment Modes" data-authors="Mathias Heltberg*, Ryan A. Kellogg*, Sandeep Krishna, Savaş Tay, Mogens H. Jensen" data-description="Oscillations and noise drive many processes in biology, but how both affect the activity of the transcription factor nuclear factor κB (NF-κB) is not understood. Here, we observe that when NF-κB oscillations are entrained by periodic tumor necrosis factor (TNF) inputs in experiments, NF-κB exhibits jumps between frequency modes, a phenomenon we call “cellular mode-hopping.” By comparing stochastic simulations of NF-κB oscillations to deterministic simulations conducted inside and outside the chaotic regime of parameter space, we show that noise facilitates mode-hopping in all regimes. However, when the deterministic system is driven by chaotic dynamics, hops between modes are erratic and short-lived, whereas in experiments, the system spends several periods in one entrainment mode before hopping and rarely visits more than two modes. The experimental behavior matches our simulations of noise-induced mode-hopping outside the chaotic regime. We suggest that mode-hopping is a mechanism by which different NF-κB-dependent genes under frequency control can be expressed at different times.">
                <img src="lib/img/thumbs/heltberg2016_thumb.jpg" alt="heltberg2016"/>
              </a>
            </li>
            <li>
              <a href="lib/ppr/kellogg2017.pdf" data-largesrc="lib/img/kellogg2017.jpg" data-title="Cellular Decision Making by Non-Integrative Processing of TLR Inputs" data-authors="Ryan A. Kellogg, Chengzhe Tian, Martin Etzrodt, Savaş Tay" data-description="Cells receive a multitude of signals from the environment, but how they process simultaneous signaling inputs is not well understood. Response to infection, for example, involves parallel activation of multiple Toll-like receptors (TLRs) that converge on the nuclear factor κB (NF-κB) pathway. Although we increasingly understand inflammatory responses for isolated signals, it is not clear how cells process multiple signals that co-occur in physiological settings. We therefore examined a bacterial infection scenario involving co-stimulation of TLR4 and TLR2. Independent stimulation of these receptors induced distinct NF-κB dynamic profiles, although surprisingly, under co-stimulation, single cells continued to show ligand-specific dynamic responses characteristic of TLR2 or TLR4 signaling rather than a mixed response, comprising a cellular decision that we term “non-integrative” processing. Iterating modeling and microfluidic experiments revealed that non-integrative processing occurred through interaction of switch-like NF-κB activation, receptor-specific processing timescales, cell-to-cell variability, and TLR cross-tolerance mediated by multilayer negative feedback.">
                <img src="lib/img/thumbs/kellogg2017_thumb.jpg" alt="kellogg2017"/>
              </a>
            </li>
            <li>
              <a href="lib/ppr/kellogg2015.pdf" data-largesrc="lib/img/kellogg2015.jpg" data-title="Noise facilitates transcriptional control under dynamic inputs" data-authors="Ryan A. Kellogg, Savaş Tay" data-description="Cells must respond sensitively to time-varying inputs in complex signaling environments. To understand how signaling networks process dynamic inputs into gene expression outputs and the role of noise in cellular information processing, we studied the immune pathway NF-κB under periodic cytokine inputs using microfluidic single-cell measurements and stochastic modeling. We find that NF-κB dynamics in fibroblasts synchronize with oscillating TNF signal and become entrained, leading to significantly increased NF-κB oscillation amplitude and mRNA output compared to non-entrained response. Simulations show that intrinsic biochemical noise in individual cells improves NF-κB oscillation and entrainment, whereas cell-to-cell variability in NF-κB natural frequency creates population robustness, together enabling entrainment over a wider range of dynamic inputs. This wide range is confirmed by experiments where entrained cells were measured under all input periods. These results indicate that synergy between oscillation and noise allows cells to achieve efficient gene expression in dynamically changing signaling environments.">
                <img src="lib/img/thumbs/kellogg2015_thumb.jpg" alt="kellogg2015"/>
              </a>
            </li>
            <li>
              <a href="lib/ppr/kellogg2014.pdf" data-largesrc="lib/img/kellogg2014.png" data-title="High-throughput microfluidic single-cell analysis pipeline for studies of signaling dynamics" data-authors="Ryan A. Kellogg, Rafael Gómez-Sjöberg, Anne A Leyrat, Savaş Tay" data-description="Time-dependent analysis of dynamic processes in single live cells is a revolutionary technique for the quantitative studies of signaling networks. Here we describe an experimental pipeline and associated protocol that incorporate microfluidic cell culture, precise stimulation of cells with signaling molecules or drugs, live-cell microscopy, computerized cell tracking, on-chip staining of key proteins and subsequent retrieval of cells for high-throughput gene expression analysis using microfluidic quantitative PCR (qPCR). Compared with traditional culture dish approaches, this pipeline enhances experimental precision and throughput by orders of magnitude and introduces much-desired new capabilities in cell and fluid handling, thus representing a major step forward in dynamic single-cell analysis. A combination of microfluidic membrane valves, automation and a streamlined protocol now enables a single researcher to generate 1 million data points on single-cell protein localization within 1 week, in various cell types and densities, under 48 predesigned experimental conditions selected from different signaling molecules or drugs, their doses, timings and combinations.">
                <img src="lib/img/thumbs/kellogg2014_thumb.png" alt="kellogg2014"/>
              </a>
            </li>
            <li>
              <a href="lib/ppr/kellogg2015b.pdf" data-largesrc="lib/img/kellogg2015b.png" data-title="Digital signaling decouples activation probability and population heterogeneity" data-authors="Ryan A. Kellogg, Chengzhe Tian, Tomasz Lipniacki, Stephen R Quake, Savaş Tay" data-description="Digital signaling enhances robustness of cellular decisions in noisy environments, but it is unclear how digital systems transmit temporal information about a stimulus. To understand how temporal input information is encoded and decoded by the NF-κB system, we studied transcription factor dynamics and gene regulation under dose- and duration-modulated inflammatory inputs. Mathematical modeling predicted and microfluidic single-cell experiments confirmed that integral of the stimulus (or area, concentration × duration) controls the fraction of cells that activate NF-κB in the population. However, stimulus temporal profile determined NF-κB dynamics, cell-to-cell variability, and gene expression phenotype. A sustained, weak stimulation lead to heterogeneous activation and delayed timing that is transmitted to gene expression. In contrast, a transient, strong stimulus with the same area caused rapid and uniform dynamics. These results show that digital NF-κB signaling enables multidimensional control of cellular phenotype via input profile, allowing parallel and independent control of single-cell activation probability and population heterogeneity.">
                <img src="lib/img/thumbs/kellogg2015b_thumb.png" alt="kellogg2015b"/>
              </a>
            </li>
          </ul>
        </div>
      </div>
    </div>


  
  <!-- Footer: Social links -->
  <div class="section" id="footer">
    <div class="container">
      <div class="row">
        <div class="col-sm-12">
          <ul class="brands">
            <li><a href="https://www.facebook.com/rakellogg"><i class="fa fa-facebook"></i></a></li>
            <li><a href="https://twitter.com/ryan_kellogg"><i class="fa fa-twitter"></i></a></li>
            <li><a href="https://www.linkedin.com/in/rkellogg/"><i class="fa fa-linkedin"></i></a></li>
          </ul>

          <p>© Ryan Kellogg 2017 All right reserved.</p>
        </div>
      </div>
    </div>
  </div> 


  <!-- Javascript Imports -->
  <script src="js/bootstrap.min.js"></script>
  <script src="js/jquery.min.js"></script>
  <script src="js/modernizr.custom.js"></script>
  <script src="js/grid.js"></script>
  <script src="js/wow.min.js"></script>
  <script src="js/jquery.simple-text-rotator.min.js"></script>
  <script src="js/main.js"></script>
</body>
</html>